hackclub.com.yaml

```diff
@@ -2855,3 +2859,8 @@ syria:
   - ttl: 600
     type: CNAME
     value: hackclub-syria.github.io.
+    
+astra:
+  - ttl: 600
+    type: CNAME
+    value: astra-collective.github.io.
```

Comment:
Delete this redundant entry.

Reply:
ok...other two entries had probelems in it so i closed them.

Reply:
will make new correct PR